spec/support/platform_helpers.rb

```diff
@@ -9,7 +9,11 @@ def jruby?
     RUBY_ENGINE == 'jruby'.freeze
   end
 
+  def truffleruby?
+    RUBY_ENGINE == 'truffleruby'.freeze
+  end
+
   def supports_fork?
```

Comment:
In general, a better way to check that is `Process.respond_to?(:fork)`:
https://github.com/oracle/truffleruby/blob/master/doc/user/compatibility.md#fork

Reply:
Fixed in 29e73f59fc910ee1113696f86aa02fd95e01fff3, thanks for the hint :)